src/aliceVision/sfmDataIO/viewIO.hpp

```diff
@@ -91,5 +91,16 @@ std::shared_ptr<camera::IntrinsicBase> getViewIntrinsic(
 */
 std::vector<std::string> viewPathsFromFolders(const sfmData::View& view, const std::vector<std::string>& folders);
 
+
+/*
+* @brief Allows to detect if an image is part of an image sequence from the analysis of its filename
```

Comment:
You can add here an explanation of the expected pattern

Reply:
Thanks.
I have made the changes.